src/index.ts

```diff
@@ -42,26 +41,24 @@ export interface IgnorerOptions {
 }
 
 export interface LogIgnorerOptions extends StrykerOptions {
-    logignore: IgnorerOptions
+    logignore?: IgnorerOptions
 }
 
 /*
  * LogIgnorer provides functionality to ignore log calls for provides logger object names.
  */
 export class LogIgnorer implements Ignorer {
-    public static inject = [commonTokens.logger, commonTokens.options] as const
-    protected readonly options: LogIgnorerOptions
+    public static inject = [commonTokens.options] as const
+    protected readonly options: IgnorerOptions | undefined
```

Comment:
```suggestion
    protected readonly options?: IgnorerOptions
```
Could be shortend using question mark syntax